auth.js

```diff
@@ -51,6 +51,7 @@ auth.post('/users', async ctx => {
     }
     if (success) {
       ctx.login(user);
+      ctx.request.flash('loggedin', "Welcome to Coffeegrams, " + user.name);
```

Comment:
Why not just `success` instead of `loggedin`? `type` is just for indicating if it's success or error usually, no reason to create a bunch of custom types ;)